spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/AbstractConnectionFactoryConfigurer.java

```diff
@@ -40,11 +40,11 @@ protected AbstractConnectionFactoryConfigurer(RabbitProperties properties) {
 		this.rabbitProperties = properties;
 	}
 
-	protected final ConnectionNameStrategy getConnectionNameStrategy() {
+	public final ConnectionNameStrategy getConnectionNameStrategy() {
 		return this.connectionNameStrategy;
 	}
 
-	protected final void setConnectionNameStrategy(ConnectionNameStrategy connectionNameStrategy) {
+	public final void setConnectionNameStrategy(ConnectionNameStrategy connectionNameStrategy) {
```

Comment:
This is needed [here](https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/pull/323/files#diff-91292975db6a0350e86f703102cb7596bc9f5741b8372ef5fcadf6b156c09ce0R71).

Reply:
Sorry, I got a bit carried away during my polishing. As we haven't shipped the configurer, I've corrected the polishing in 069074e49032b3a7895a3ccfeae4808777d5f6c5.